networks/rpc/utils.go

```diff
@@ -247,3 +247,38 @@ func sanitizeTimeouts(timeouts HTTPTimeouts) HTTPTimeouts {
 
 	return timeouts
 }
+
+func SchemeToString(scheme ServerScheme) string {
+	switch scheme {
+	case HTTPServer:
+		return "HTTP"
+	case FastHTTPServer:
+		return "FastHTTP"
+	case WSServer:
+		return "WS"
+	case FastWSServer:
+		return "FastWS"
+	case GRPCServer:
+		return "GRPC"
+	case InProcServer:
+		return "InProc"
+	case IPCServer:
+		return "IPC"
+
```

Comment:
remove the line plz

Reply:
Do you mean the below?
```
case HTTPServer:
    return "HTTP"
...
```
If so, the line insert seems to be enforced by gofmt.

Reply:
Oh, I checked what you mean. Removed. Thank you